samples/msgext-ai-sentiment-analysis/nodejs/infra/botRegistration/readme.md

```diff
@@ -0,0 +1 @@
+The `azurebot.bicep` module is provided to help you create Azure Bot service when you don't use Azure to host your app. If you use Azure as infrastrcture for your app, `azure.bicep` under infra folder already leverages this module to create Azure Bot service for you. You don't need to deploy `azurebot.bicep` again.
```

Comment:
Why we need this readme?

Reply:
Auto generated, Not sure to keep or remove.

Reply:
Build Pile Line Executed: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5935565374